lale/lib/sklearn/stacking_classifier.py

```diff
@@ -0,0 +1,260 @@
+                        ],
+                    },
+                    "description": "Base estimators which will be stacked together. Each element of the list is defined as a tuple of string (i.e. name) and an estimator instance. An estimator can be set to ‘drop’ using set_params.",
+                },
+                "final_estimator": {
+                    "anyOf": [{"laleType": "operator"}, {"enum": [None]}],
+                    "default": None,
+                    "description": "A classifier which will be used to combine the base estimators. The default classifier is a 'LogisticRegression'",
+                },
+                "cv": {
+                    "XXX TODO XXX": "int, cross-validation generator or an iterable, optional",
+                    "description": "Determines the cross-validation splitting strategy",
+                    "anyOf": [
+                        {
+                            "type": "integer",
+                            "minimumForOptimizer": 3,
+                            "maximumForOptimizer": 4,
+                            "distribution": "uniform",
+                        },
+                        {"enum": [None]},
+                    ],
+                    "default": None,
+                },
+                "stack_method": {
+                    "description": "Methods called for each base estimator. If ‘auto’, it will try to invoke, for each estimator, 'predict_proba', 'decision_function' or 'predict' in that order. Otherwise, one of 'predict_proba', 'decision_function' or 'predict'. If the method is not implemented by the estimator, it will raise an error.",
+                    "default": "auto",
+                    "enum": ["auto", "predict_proba", "decision_function", "predict"],
+                },
+                "n_jobs": {
+                    "anyOf": [{"type": "integer"}, {"enum": [None]}],
+                    "default": None,
+                    "description": "The number of jobs to run in parallel for ``fit``.",
+                },
+                "passthrough": {
+                    "type": "boolean",
+                    "default": False,
+                    "description": "When False, only the predictions of estimators will be used as training data for 'final_estimator'. When True, the 'final_estimator' is trained on the predictions as well as the original training data.",
+                },
+            },
+        },
+    ],
+}
+_input_fit_schema = {
+    "description": "Fit the estimators.",
+    "type": "object",
+    "required": ["X", "y"],
+    "properties": {
+        "X": {
+            "type": "array",
+            "items": {
+                "type": "array",
+                "items": {"type": "number"},
+            },
+            "description": "Training vectors, where n_samples is the number of samples and n_features is the number of features.",
+        },
+        "y": {
+            "type": "array",
+            "items": {"type": "number"},
```

Comment:
The schema for `y` might be too restrictive, because in practice, class labels are often not numbers but strings or booleans. Instead, I'd make it an `anyOf`, like in [DecisionTreeClassifier](https://github.com/IBM/lale/blob/master/lale/lib/sklearn/decision_tree_classifier.py#L214-L221).